contrib/upldrel/entrypoint.sh

```diff
@@ -4,7 +4,14 @@ set -e
 
 source /usr/local/bin/lib_entrypoint.sh
 
-req_env_var GCPJSON_FILEPATH GCPNAME GCPPROJECT BUCKET FROM_FILEPATH TO_FILENAME ALSO_FILENAME
+download_url=$(curl -s https://api.github.com/repos/go-swagger/go-swagger/releases/latest | \
+  jq -r '.assets[] | select(.name | contains("'"$(uname | tr '[:upper:]' '[:lower:]')"'_amd64")) | .browser_download_url')
```

Comment:
I'm not sure if `jq` is available inside this container, but it could be added.

Reply:
Also this is going to execute for EVERY file upload, including each and every release archive.  Perhaps it would be wise to check if TO_FILENAME contains `yaml`, and if not skip this?